docs/api-query.md

```diff
@@ -45,14 +45,23 @@ clarity and brevity.
 Create a new query object to process the given set of files or directories.
 
 ```python
-Query(*paths: Union[str, List[str]], filename_matcher: FilenameMatcher)
+Query(
+  *paths: Union[str, List[str]],
+  python_version: int,
+  filename_matcher: FilenameMatcher,
+)
 ```
 
 * `*paths` - Accepts either individual file or directory paths (relative to the current
   working directory), or lists of paths, for any positional argument given.
   Defaults to the current working directory if no arguments given.
 
-* `*filename_matcher*` - A callback which returns whether a given filename is
+* `python_version` - The 'major' python version of the files to be refactored, i.e. `2`
```

Comment:
Can you add asterisks back on the param names please?

Reply:
sorry, I thought maybe they were intended as bold formatting but they display as literal asterisks (and it's not an *args param)

will put them back

also I will reorder docs and code to put my new arg after the existing one, that would be safer

Reply:
done in ff8c0d4273be7f35141d04e407509717f53c34e1

Reply:
@jreese ?